### ISSUE TITLE
[Bug] - insomnia hangs when view raw response of binary file

### ISSUE THREAD

**Issue by soleverlee** [getinsomnia/insomnia#1127]
- Insomnia Version: Version 6.0.2 (6.0.2.2311)
- Operating System: Mac os 10.13.6 (17G65)

## Details

I use insomnia to call an api which returns a zip binary file, it works well and I saved the response body; when I change the response type to "Raw", it hangs without any feedback, even if I restart the application, therefore I have no chance to change it back.

So is this a bug? And could you please tell me how to fix this mannually like editing the configuration or something else?
<img width="1154" alt="bug" src="https://user-images.githubusercontent.com/6427851/44385081-f0293800-a550-11e8-987d-94f9bc054106.png">

**Comment by soleverlee** [comment]
It worked when I mannually changed settings under ~/Library/Application Support/Insomnia/insomnia.RequestMeta.db， ie. change "raw" to "friendly".

**Comment by carmineesposito73** [comment]
I don't know when, but the issue disappeared

$ snap list insomnia 
Name      Version  Rev  Tracking  Publisher    Notes
insomnia  6.0.2    19   stable    gschier1990  -

$ cat /etc/*rel*
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.10

**Comment by enricovisentini** [comment]
I experienced the same issue with the same version, while previewing a 2.2MB pdf binary

**Comment by avwhite** [comment]
I am seeing the same issue already at a couple of hundreds of kilobytes of binary data. I didn't realize I could get around this by setting the view mode to preview or source, so that is nice and basically fixes the problem for me.

**Comment by maximeMD** [comment]
I am experiencing the same issue with a JPEG file data. I noticed that the app stops freezing and displays the raw data, after a **few minutes** .
Wouldn't it be great to have some spinner, or at least avoid freezing the app ?
FTR, Insomnia when 'loading' uses 18% of my Intel Core i7 3370K CPU (3.5Ghz)

Thank you for your great work anyway 😄

**Comment by ypkkhatri** [comment]
I'm also facing same issue in Ubuntu 19.04, while opening JPEG file in raw.

Thank you

**Comment by Shedon** [comment]
same for me

**Comment by AryanshMahato** [comment]
I am having this same issue

**Comment by heiybb** [comment]
Similar issue here while opening PDF file in raw, when I clicked the Raw option at the Response section, the app got stuck, then I have to restart it, but it won't work, everytime launching the app, it will get stuck at the previous page I closed before. The CPU usage will also raise up the same time(20% usage for 8750H).
Around 5 mins later, the app recovered from the frozen status and automatically jumped to another request, but once I clicked the request (that crash the app before), it will get stuck again.

How I get rid of the infinite boot stuck loop:
1.Wait it recovery and delete the problem request(that crash the app)
2.Go the insomnia's data dir and delete the problem request manually from `insomnia.Request.db`
(For windows user it locates at `C:\Users\yourusername\AppData\Roaming\Insomnia`)
